service/controller/app/v1/resource/chart/desired.go

```diff
@@ -73,11 +74,11 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 func generateAnnotations(input map[string]string) map[string]string {
 	annotations := map[string]string{}
 
-	// ForceHelmUpgrade has been set for this app CR so this needs to be passed
-	// on to the chart CR.
-	val, ok := input[annotation.ForceHelmUpgrade]
-	if ok {
-		annotations[annotation.ForceHelmUpgrade] = val
+	for k, val := range input {
```

Comment:
```suggestion
	for k, v := range input {
```

Let's try and be consistent with var naming. This is shorter but I'd also be fine with `key` and `val`.

Reply:
Agreed, shorther version I would like.